src/udev/udev-builtin-input_id.c

```diff
@@ -47,6 +47,7 @@
 /* available as of kernel 3.11 */
 #ifndef BTN_DPAD_UP
 #define BTN_DPAD_UP 0x220
+#define BTN_DPAD_RIGHT 0x223
```

Comment:
hmm, we actually have src/basic/missing.h that covers all these little things thatmight be missing from headers we include... But let's merge this here for now.